mParticle-RevealMobile/MPKitRevealMobile.m

```diff
@@ -66,8 +90,25 @@ - (void)start {
         /*
             Start your SDK here. The configuration dictionary can be retrieved from self.configuration
          */
+        NSString *appKey = self.configuration[@"apiKey"];
+        // Get a reference to the SDK object
+       self.sdk = [Reveal sharedInstance];
+       //self.sdk.locationManager = [RVLAltLocatioManager new];
+       
+       // Turn on debug logging, not for production
+       self.sdk.debug = YES;
+       self.sdk.delegate = self;
+       
+       [self.sdk setupWithAPIKey: appKey andServiceType: RVLServiceTypeProduction];
 
         _started = YES;
+    
+        // Once the config values are set, start the SDK.
+        // The SDK will contact the server for further config info
+        // and start monitoring for beacons.
+        [self.sdk start];
+
+        NSLog( @"MPKitRevealMobile start completed" );
```

Comment:
Do we need this log message? The mParticle SDK already broadcasts a notification when kits are started